.github/workflows/llvm.yml

```diff
@@ -17,17 +17,19 @@ jobs:
       matrix:
         include:
           - llvm_version: "13.0.0"
-            llvm_ubuntu_version: "20.04"
+            llvm_filename: "clang+llvm-13.0.0-x86_64-linux-gnu-ubuntu-20.04.tar.xz"
           - llvm_version: "14.0.0"
-            llvm_ubuntu_version: "18.04"
+            llvm_filename: "clang+llvm-14.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz"
           - llvm_version: "15.0.6"
-            llvm_ubuntu_version: "18.04"
+            llvm_filename: "clang+llvm-15.0.6-x86_64-linux-gnu-ubuntu-18.04.tar.xz"
           - llvm_version: "16.0.3"
-            llvm_ubuntu_version: "22.04"
+            llvm_filename: "clang+llvm-16.0.3-x86_64-linux-gnu-ubuntu-22.04.tar.xz"
           - llvm_version: "17.0.6"
-            llvm_ubuntu_version: "22.04"
+            llvm_filename: "clang+llvm-17.0.6-x86_64-linux-gnu-ubuntu-22.04.tar.xz"
           - llvm_version: "18.1.4"
-            llvm_ubuntu_version: "18.04"
+            llvm_filename: "clang+llvm-18.1.4-x86_64-linux-gnu-ubuntu-18.04.tar.xz"
```

Comment:
**praise:** I like this change. It's a bit more boilerplate, but also more expressive than the previous version. I would probably prefer this even if it wasn't necessitated by a change in the naming scheme.